lib/ui/anytime_podcast_app.dart

```diff
@@ -246,11 +246,14 @@ class _AnytimeHomePageState extends State<AnytimeHomePage> with WidgetsBindingOb
     final pager = Provider.of<PagerBloc>(context);
     final searchBloc = Provider.of<EpisodeBloc>(context);
     final backgroundColour = Theme.of(context).scaffoldBackgroundColor;
+    final isLight = Theme.of(context).brightness == Brightness.light;
 
     return AnnotatedRegion<SystemUiOverlayStyle>(
       value: SystemUiOverlayStyle(
-        statusBarIconBrightness: Theme.of(context).brightness == Brightness.light ? Brightness.dark : Brightness.light,
-        systemNavigationBarColor: Theme.of(context).bottomAppBarColor,
+        statusBarIconBrightness: isLight ? Brightness.dark : Brightness.light,
+        systemNavigationBarColor: isLight
+            ? Theme.of(context).primaryColorDark
+            : Theme.of(context).bottomAppBarColor,
```

Comment:
What about using `primaryColorLight` when theme is `dark` ?

Reply:
That works too, this PR was open 3 months ago, It seems the maintainer has changed this code during this time so probably this code is irrelevant today.

Reply:
Hi,

I have updated the code to work with the theme changes in Flutter 2.8, so you're right, this is largely redundant now. Do you have any further issues with this @ademar111190 ? If not I will close this PR. Thanks.

Reply:
No